cpp/log/sqlite_db-inl.h

```diff
@@ -164,24 +168,7 @@ typename Database<Logged>::WriteResult SQLiteDB<Logged>::CreateSequencedEntry_(
 
   int ret = statement.Step();
   if (ret == SQLITE_CONSTRAINT) {
-    sqlite::Statement s2(
-        db_, "SELECT sequence, hash FROM leaves WHERE sequence = ?");
-    s2.BindUInt64(0, logged.sequence_number());
-    if (s2.Step() == SQLITE_ROW) {
-      std::string existing_hash;
-      s2.GetBlob(1, &existing_hash);
-
-      if (hash == existing_hash) {
-        return this->OK;
```

Comment:
We need to keep this case, it's just the second `SELECT` at lines 181-184 that need to go.